tools/project_manager/editproject.php

```diff
@@ -560,4 +573,18 @@ public function normalize_spaces()
         $this->project->clearance = preg_replace('/\s+/', ' ', trim($this->project->clearance));
         $this->project->extra_credits = preg_replace('/\s+/', ' ', trim($this->project->extra_credits));
     }
+
+    private function serialize($value)
+    {
```

Comment:
Could not redefining a function with the same name lead to confusion for future generations?

Reply:
It's a class function so it isn't overriding the built-in (not that PHP supports that like python does). The only way to call this function is to use the object.